modules/gapi/src/executor/gexecutor.cpp

```diff
@@ -117,11 +117,36 @@ void cv::gimpl::GExecutor::initResource(const ade::NodeHandle &orig_nh)
         // Constructed on Reset, do nothing here
         break;
 
+    // FIXME: What about GOPAQUE here??
```

Comment:
? :)

Reply:
Added in https://github.com/opencv/opencv/pull/16964, removed this FIXME